AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -3015,4 +2948,4 @@ public void onProgressUpdate(String message) {
         /** Always open reviewer (keyboard shortcut) */
         SKIP_STUDY_OPTIONS
     }
-}
+}
```

Comment:
nit: spacing change (newline removed)

Reply:
yeah this is likely an editor setting - historically we use newlines at the end of files, but regardless if a line does not matter for the PR it should not be touched
```suggestion
}
```